validator/rpc/standard_api_test.go

```diff
@@ -724,23 +728,60 @@ func TestServer_ListFeeRecipientByPubkey(t *testing.T) {
 			wantErr: false,
 		},
 		{
-			name: "empty settings",
+			name: "Happy Path Test Cached",
+			args: &validatorserviceconfig.ProposerSettings{
+				ProposeConfig: map[[48]byte]*validatorserviceconfig.ProposerOption{
+					bytesutil.ToBytes48(byteval): {
+						FeeRecipient: common.HexToAddress("0x046Fb65722E7b2455012BFEBf6177F1D2e9738D9"),
```

Comment:
The same hex magic string is used 4 times here. Can you extract it to a local constant?

Actually it might make sense to create local constants for all hex values because it's hard to see if values are the same or not. For example, in `TestServer_SetFeeRecipientByPubkey` the difference between `"0x055Fb65722e7b2455012Bfebf6177f1d2e9738d7"` and `"0x055Fb65722E7b2455012BFEBf6177F1D2e97387"` is so small, it's easy to get confused.